### PR TITLE
Version 1.10.1 patch release

### PR DESCRIPTION
**Pull request by ewels**
See https://github.com/nf-core/tools/pull/707 for changes.

`dev` to `master` for the 1.10.1 patch release 🚀

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from setuptools import setup, find_packages
 import sys
 
-version = "1.10"
+version = "1.11dev"
```
Comment (apeltzer) [inline setup.py:6]: 1.10.2

Comment (ewels) [reply setup.py:6]: 😱

Comment (ewels) [reply setup.py:6]:
```suggestion
version = "1.10.1"
```

Comment (drpatelh) [reply setup.py]: Good spot @apeltzer !! I think @ewels needs to give you some of his commits now 🙁

**Pull request merged**